joomla/joomla-cms
# Using the "Special:MyLanguage" tag for links pointing to docs.joomla.org

### Summary of Changes

Using the "Special:MyLanguage" tag for links pointing to docs.joomla.org

### Testing Instructions

- install: https://github.com/zero-24/joomla-cms/archive/mylanguage.zip
- confirm that links to docs.joomla.org point to the localised version if possible
- like the link for the template manager ...
- ... or the readme links in this repo

### Expected result

Links to docs.joomla.org point to the localised version if possible

### Actual result

Links to docs.joomla.org points to the en-GB version even if there is a localised version

### Documentation Changes Required

None.
